src/sass/flexboxgrid.scss

```diff
@@ -65,6 +65,15 @@ $offset-properties: 'margin-left' !default;
 
 @mixin columns($selector: $column-selector, $properties: $column-properties, $columns: $column-count, $queries: $queries) {
   @each $size in $sizes {
+
+    $iterator-selector: #{$selector}#{$block-delimiter}#{$size};
+
+    @for $i from 0 through $columns {
+      #{if($i == 0, $iterator-selector, #{$iterator-selector}#{$modifier-delimiter}#{$i})} {
```

Comment:
We're using this branch in our project and getting compile errors at this line:

[gulp-sass] property "#{if($i == 0, $iterator-selector, #{$iterator-selector}#{$modifier-delimiter}#{$i}" must be followed by a ':' on line 72 in node_modules/flexboxgrid/src/sass/flexboxgrid.scss